source/minar.cpp

```diff
@@ -51,11 +48,11 @@ extern "C" {
 #define ytTraceDispatch(...) do{}while(0)
 #endif
 
-#ifndef __MINAR_NO_RUNTIME_WARNINGS__
+#if YOTTA_CFG_MINAR_NO_RUNTIME_WARNINGS
```

Comment:
Will it always be defined? Else, this will issue a warning with `-Wundef` on gcc. I don't know what the ARM mbed policy on warnings is.

Apart from that, +1 from me on the code.